src/core/net/mdns.hpp

```diff
@@ -1440,14 +1428,17 @@ class Core : public InstanceLocator, private NonCopyable
 
     // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-    class CacheTimerContext : public TimerContext
+    class CacheTimerContext : public InstanceLocator
     {
     public:
         CacheTimerContext(Instance &aInstance);
-        TxMessage &GetQueryMessage(void) { return mQueryMessage; }
+        TimeMilli     GetNow(void) const { return mNextFireTime.GetNow(); }
```

Comment:
Same as above.

Reply:
Same here. For example used in `PrepareQuery(CacheTimerContext &aContext)`